AEPConcierge/Sources/Theme/ConciergeTheme.swift

```diff
@@ -0,0 +1,179 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import SwiftUI
+import UIKit
+import AEPServices
+
+// MARK: - Theme Configuration Types
+
+/// Helper CodingKey for decoding dynamic CSS variable names
+private struct DynamicCodingKey: CodingKey {
+    var stringValue: String
+    var intValue: Int?
+    
+    init?(stringValue: String) {
+        self.stringValue = stringValue
+        self.intValue = nil
+    }
+    
+    init?(intValue: Int) {
+        self.stringValue = String(intValue)
+        self.intValue = intValue
+    }
+}
+
+/// Main ConciergeTheme structure that consolidates all theme configuration
+/// Maps to the web styleConfiguration format for cross platform compatibility
+/// 
+/// JSON Structure Mapping:
+/// - `metadata` -> metadata
+/// - `behavior` -> behavior
+/// - `disclaimer` -> disclaimer
+/// - `text` -> text/copy (with nested structure matching web dot-notation keys)
+/// - `arrays.welcome.examples` -> welcomeExamples
+/// - `arrays.feedback.positive.options` -> feedbackPositiveOptions
+/// - `arrays.feedback.negative.options` -> feedbackNegativeOptions
+/// - `assets` -> assets
+/// - `theme` -> colors, layout, typography, components (CSS variables mapped to semantic groups)
+public struct ConciergeTheme: Codable {
```

Comment:
this file seems like it got included twice - once in the `Theme` directory and once in the `Models` directory.  let's pick one to keep :)